src/controllers/v1/certificate/index.ts

```diff
@@ -50,41 +52,48 @@ export class CertificateController extends Controller {
   }
 
   /**
+   * TODO handle multiple
    * @summary insert a certificate for initialisation
    */
-  @Example<Certificate.Payload>({
-    id: '52907745-7672-470e-a803-a2f8feb52944',
-    capacity: 10,
-    co2e: 1000,
-  })
   @Post()
   @Response<BadRequest>(400, 'Request was invalid')
   @Response<ValidateError>(422, 'Validation Failed')
   @SuccessResponse('201')
-  public async post(@Body() body: Certificate.Request): Promise<Certificate.Response> {
-    this.log.info({ identity: this.identity, body })
-    const formatted = camelToSnakeJSObject(body)
+  public async postDraft(
+    @Body() { hydrogen_quantity_mwh, energy_owner }: Certificate.Payload
+  ): Promise<Certificate.Response> {
+    this.log.info({ identity: this.identity, energy_owner })
+
+    const { address: hydrogen_owner } = await this.identity.getMemberBySelf()
+
+    // TODO - handle error if alias not found
+    // const { address: energy_owner_address } = await this.identity.getMemberByAlias(energy_owner)
 
     return {
       message: 'ok',
-      result: await this.db.insert('certificate', formatted),
+      result: await this.db.insert('certificate', {
+        hydrogen_quantity_mwh,
+        hydrogen_owner,
+        energy_owner: '5HpG9w8EBLe5XCrbczpwq5TSXvedjrBGCwqxK1iQ7qUsSWFc',
+      }),
     }
   }
 
   /**
    *
    * @summary Lists all local certificates
-   * TODO - combine where so with all possible options e.g.
+   * TODO - combine [where] so with all possible options e.g.
    *   columns so it's not a default and will be rendered in swagger
    */
-
   @Get('/')
+  @Response<NotFound>(404, '<item> not found')
   public async getAll(@Query() createdAt?: DATE): Promise<Certificate.Response> {
     const where: Record<string, number | string | DATE> = {}
     if (createdAt) where.created_at = createdAt
 
     return {
       message: 'ok',
+      // TODO - transform hydrogen_owner and energy_owner to aliases
```

Comment:
Still needs fixing prior to merging

Reply:
This is also, just making sure got all covered so replied can be ignored which do not ask.